### PR TITLE
[SDESK-1798] Refactor purging the audit collection

### PR DESCRIPTION
**Pull request by marwoodandrew**

### CODE REVIEW

File: superdesk/data_updates/00003_20170814-114652_audit.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+
+class DataUpdate(DataUpdate):
+
+    resource = 'audit'
+
+    def forwards(self, mongodb_collection, mongodb_database):
+        for audit in mongodb_collection.find({}):
+            audit_id = get_resource_service(self.resource)._extract_doc_id(audit.get('extra'))
+            print(mongodb_collection.update({'_id': audit.get(config.ID_FIELD)},
```
Comment (petrjasek) [inline superdesk/data_updates/00003_20170814-114652_audit.py]: imo would make sense to add an index on audit_id once it gets populated

Comment (marwoodandrew) [inline superdesk/data_updates/00003_20170814-114652_audit.py]: @petrjasek Yes we've been tuning it on our UAT system, It's just a painfully slow process! 

Do you think the index should be coded in the resource. or just create it in Mongo externally?

Comment (petrjasek) [inline superdesk/data_updates/00003_20170814-114652_audit.py]: I would rather create it here once the migration is done

Comment (marwoodandrew) [inline superdesk/data_updates/00003_20170814-114652_audit.py]: Yep, no problem!

**Pull request merged**